README.md

```diff
@@ -160,4 +160,4 @@ Another example of equivalence is as follows:
 -  `self.packages.x86_64-linux.hello`
 
 [age]: https://github.com/FiloSottile/age "age"
-[agenix]: https://github.com/ryantm/agenix "agenix"
\ No newline at end of file
+[agenix]: https://github.com/ryantm/agenix "agenix"
```

Comment:
This is so strange, it shows a diff but the lines are the same?

Aside from my confusion about that LGTM